api/cpp/esp-idf/slint/README.md

```diff
@@ -11,81 +11,110 @@ It has been tested on ESP32-S3 devices.
 
 ![Screenshot](https://user-images.githubusercontent.com/959326/260754861-e2130cce-9d2b-4925-9536-88293818ac3e.jpeg)
 
-## Prerequisites
+## Getting Started
 
-In order to compile Slint, you need a working [Rust toolchain for your device](https://esp-rs.github.io/book/installation/index.html).
-Follow the instructions from <https://github.com/esp-rs/espup#installation> to set it up.
+### Prerequisites
 
-You can then either set the esp toolchain as the default toolchain with `rustup` or create a `rust-toolchain.toml` file in the root directory of your project with the following contents:
+* Install the Espressif IoT Development Framework](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/index.html) and open a terminal or command prompt with the environment set up.
+On Windows, follow the [Using the Command Prompt](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/windows-setup.html#using-the-command-prompt) instructions, on macOS and Linux, follow the
+[Set up the Environment Variables](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/linux-macos-setup.html#step-4-set-up-the-environment-variables) instructions.
+* Install a [Rust toolchain for your device](https://esp-rs.github.io/book/installation/index.html).
+* Install the [Rust toolchains for Espressif SoCs](https://esp-rs.github.io/book/installation/index.html).
+  
+### Hello World
 
+The following steps will guide from the a bare-bones esp-idf "hello_world" to a GUI with Slint.
+
+1. Start by creating a new project:
+```bash
+idf.py create-project slint-hello-world
```

Comment:
before running idf.py,  i need to do `. $HOME/esp/esp-idf/export.fish`  on the current shell.  But i suppose you assume this explaination is not required because it goes in your steps to install the toolchain.
I think i'd recap that you need to be in the ternminal that has the env variable setup.
But it's also fine to leave out.

Reply:
I was thinking about this, too, but I left it out because that only applies to macOS and Linux, but on Windows it's different. The text in line 19 covers that, basically.